src/meshing.md

```diff
@@ -52,6 +52,30 @@ the mobius-strip miniapp.
 The `klein-bottle` and `klein-donut` meshes in the
 `data` directory were generated with this miniapp.
 
+### Toroid
+
+This miniapp generates two types of toroidal volume meshes; one with
+triangular cross sections and one with square cross sections.
+
+![](img/examples/toroid-wedge.png)
+
+A wide variety of toroidal meshes can be generated by varying the
+amount of twist as well as the major and minor radii and other
+variables.  The `toroid-wedge` and `toroid-hex` meshes in the `data` directory
+were generated with this miniapp.
+
+### Extruder
+
+This miniapp creates higher dimensional meshes from lower dimensional meshes
+by extrusion.
+
+![](img/examples/extruded-star.png)
+
+Simple coordinate transformations can also be applied if desired.  The initial
+mesh can be 1D or 2D. 1D meshes can be extruded in the y-direction first and
+then in the z-direction.  2D meshes can be triangular, quadrilateral, or
+contain both element types.
+
```

Comment:
Do we want to add an example of the low-order refinement here? If so, I can write something up